packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -51,7 +51,7 @@
 
     const explorerUrl = getOfficialExplorerUrl($activeProfile?.network?.id)
 
-    $: isImplicitAccount = isImplicitAccountOutput(selectedOutput.output as CommonOutput)
+    $: isImplicitAccount = isImplicitAccountOutput(selectedOutput?.output as CommonOutput)
```

Comment:
Why the `?`, there will always be an output according to the SDK types, and even if there wasn't, this will throw an error as `isImplicitAccountOutput` try to read one of it's properties:

```ts
export function isImplicitAccountOutput(output: CommonOutput): boolean {
    return (
        output?.type === OutputType.Basic &&
        output.unlockConditions.length === 1 &&
        (
            output.unlockConditions.find(
                (cmnOutput) => cmnOutput.type === UnlockConditionType.Address
            ) as AddressUnlockCondition
        )?.address.type === AddressType.ImplicitAccountCreation
    )
}
```

```suggestion
    $: isImplicitAccount = isImplicitAccountOutput(selectedOutput.output as CommonOutput)
```